File: PidginMacOSItegration/Menu.m
//
//  Menu.m
//  PidginMacOSIntegration
//
//  Created by Agalin on 27/07/2020.
//  Copyright © 2020 Agalin. All rights reserved.
//

#import <Foundation/Foundation.h>
#import "Menu.h"
#import <gtkosxapplication.h>

void set_menu(GtkWidget *menu){
    gtkosx_application_set_menu_bar(gtkosx_application_get(), GTK_MENU_SHELL(menu));
}

File: PidginMacOSItegration/Menu.swift
//
//  Menu.swift
//  PidginMacOSIntegration
//
//  Created by Agalin on 29/07/2020.
//  Copyright © 2020 Agalin. All rights reserved.
//

import Foundation

func setMenu(_ menu: UnsafeMutablePointer<GtkWidget>!) {
    gtkosx_application_set_menu_bar(gtkosx_application_get(), unsafeBitCast(menu, to: UnsafeMutablePointer<GtkMenuShell>?.self));
}
